class-pinterest-for-woocommerce.php

```diff
@@ -153,6 +153,9 @@ public function maybe_init_plugin() {
 
 			$this->define_constants();
 
+			// Load utilities manually. This is not a class so autoloader misses it.
+			require_once 'src/Utilities/Utilities.php';
```

Comment:
We should use the full path rather than relying on relative resolution.

```suggestion
			require_once __DIR__ . '/src/Utilities/Utilities.php';
```